code/__defines/skills.dm

```diff
@@ -12,25 +12,14 @@
 #define SKILL_AVERAGE  2
 #define SKILL_HARD     4
 
-#define SKILL_BUREAUCRACY   /decl/hierarchy/skill/organizational/bureaucracy
-#define SKILL_FINANCE       /decl/hierarchy/skill/organizational/finance
 #define SKILL_EVA           /decl/hierarchy/skill/general/EVA
-#define SKILL_MECH          /decl/hierarchy/skill/general/mech
-#define SKILL_PILOT         /decl/hierarchy/skill/general/pilot
 #define SKILL_HAULING       /decl/hierarchy/skill/general/hauling
+#define SKILL_PILOT         /decl/hierarchy/skill/general/pilot
 #define SKILL_COMPUTER      /decl/hierarchy/skill/general/computer
-#define SKILL_BOTANY        /decl/hierarchy/skill/service/botany
 #define SKILL_COOKING       /decl/hierarchy/skill/service/cooking
 #define SKILL_COMBAT        /decl/hierarchy/skill/security/combat
 #define SKILL_WEAPONS       /decl/hierarchy/skill/security/weapons
-#define SKILL_FORENSICS     /decl/hierarchy/skill/security/forensics
 #define SKILL_CONSTRUCTION  /decl/hierarchy/skill/engineering/construction
 #define SKILL_ELECTRICAL    /decl/hierarchy/skill/engineering/electrical
-#define SKILL_ATMOS         /decl/hierarchy/skill/engineering/atmos
-#define SKILL_ENGINES       /decl/hierarchy/skill/engineering/engines
-#define SKILL_DEVICES       /decl/hierarchy/skill/research/devices
```

Comment:
Request devices skill to stay, it will be useful in crafting

Reply:
Honestly, can simplify devices and computers into the same thing. In terms of dead space qualifications, some who can reliably hack the complicated machinery of the 26th century is probably able to assemble most basic devices given the right tools. I really don't see the need for both.